lxc/copy.go

```diff
@@ -142,6 +149,10 @@ func (c *cmdCopy) copyContainer(conf *config.Config, sourceResource string,
 			Live: stateful,
 		}
 
+		if c.flagRefresh {
+			return fmt.Errorf("Cannot use --refresh with snapshots")
```

Comment:
Error may confuse users about whether we're talking about containers that have snapshots or a container snapshot.

Maybe "--refresh can only be used with containers" instead?

Reply:
Oh and that should be using i18n.G too